product-backend/scripts/PopulateDatabase.sql

```diff
@@ -4,7 +4,6 @@ INSERT INTO Users(email,password,roleId) VALUES ('admin@kainos.com','PVGsZaWZUCN
 INSERT INTO Users(email,password,roleId) VALUES ('user@kainos.com','KqzqGgU+R/P3fOuzjm7Vzw==:A6xnQhbz4Vx2HuGl4lXwZ5U2I8iziLRFnhP5eNfIRvQ=',1);
 
 
-INSERT INTO `Bands` (name, level) VALUES ('Apprentice', 1);
```

Comment:
Band "Apprentice" should not be deleted.

Reply:
fixed :)